index.js

```diff
@@ -342,6 +342,21 @@ module.exports = {
     },
 
     /**
+     * Call this if you plan on loading TypeScript files.
+     *
+     * Supported options:
+     * @see https://github.com/TypeStrong/ts-loader/blob/master/README.md#available-options
+     *
+     * @param {object} options
+     * @return {exports}
+     */
+    enableTypeScriptLoader(options = {}) {
+        webpackConfig.enableTypeScriptLoader(options);
```

Comment:
I think `options` should be a callback - like we do with `configureBabel()`, so that we don't need to worry about merging about default options and their options.